pilot.go

```diff
@@ -85,7 +85,14 @@ func (c *chanController) OpenChannel(target *btcec.PublicKey,
 
 	// With the connection established, we'll now establish our connection
 	// to the target peer, waiting for the first update before we exit.
-	feePerVSize, err := c.server.cc.feeEstimator.EstimateFeePerVSize(3)
+	// We target 3 blocks or less in which to establish the connection,
+	// unless the user specified `cfg.Autopilot.ConfTarget`.
+	confTarget := cfg.Autopilot.ConfTarget
+	if confTarget == 0 { // The zero values for integer and floats is 0. nil is not a valid integer or float value.
+		confTarget = 3
+	}
```

Comment:
@Roasbeef @halseth another option would be to set a default value for `Autopilot.ConfTarget` [here](https://github.com/mecampbellsoup/lnd/blob/d804b519af78bcc61c35007a49b00da4333a3b95/config.go#L259-L264) and then I don't have to check `if confTarget == 0`. 

As written now, the effective default for conf. target is 3 blocks, so perhaps we should just make it explicit in `defaultCfg`.

@Roasbeef you mentioned [in the issue](https://github.com/lightningnetwork/lnd/issues/886#issuecomment-374363595) that you thought the default was 6 blocks as the target, but [this](https://github.com/mecampbellsoup/lnd/blob/208d1199efe11fe9988591867c20506065ef8c81/pilot.go#L88) seems to indicate it's 3 - correct? Do we want 3, 6, other?